libphal/phal_dump.C

```diff
@@ -149,24 +150,29 @@ std::string convertFapiRcToString(const fapi2::ReturnCode& rc)
  */
 void writeDumpFileOnChipOpFailure(char* data, size_t len, std::filesystem::path& dumpPath)
 {
-	log(level::INFO, "writng dump data to file for chip op failure");
-	std::ofstream outfile{dumpPath, std::ios::out | std::ios::binary};
-	if (!outfile.good())
-	{
-		int err = errno;
-		log(level::ERROR, "Failed to open the dump file for writing err=%d", err);
-		throw dumpError_t(exception::FILE_OPERATION_FAILED);
+	log(level::INFO,
```

Comment:
clang format for the code that you added cannot be a separate commit. If existing code changes due to clang, that has to be a separate commit